jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -543,6 +560,35 @@ public void setContinuation(Thread thread, Continuation c) {
 		thread.setContinuation(c);
 	}
 
+/*[IF JAVA_SPEC_VERSION >= 20]*/
+	public Object[] scopedValueCache() {
+		return Thread.scopedValueCache();
+	}
+
+	public void setScopedValueCache(Object[] cache) {
+		Thread.setScopedValueCache(cache);
+	}
+
+	public Object scopedValueBindings() {
+		return Thread.scopedValueBindings();
+	}
+
+	public Object findScopedValueBindings() {
+		return Thread.findScopedValueBindings();
+	}
+
+	public void setScopedValueBindings(Object scopeValueBindings) {
+		Thread.setScopedValueBindings(scopeValueBindings);
+	}
+
+	public void ensureMaterializedForStackWalk(Object obj) {
+		Thread.ensureMaterializedForStackWalk(obj);
+	}
+
+	public InputStream initialSystemIn() {
+		return initialSystemIn;
+	}
```

Comment:
Why a function description was not added for any of the Java methods?

Reply:
FYI: @JasonFengJ9

Reply:
We typically dont document these because they just forward to other methods which are documented. Duplicating comments gets out of sync the moment we need to update something

Reply:
Yeah, `Access` is a helper class to allow privileged access to classes from outside `java.lang` package in which most of the methods just forward to other methods. Added java doc for `initialSystemIn()` which returns the initial system `in` set by `System.setIn()` instead of invoking another method.